packages/math-inline/configure/src/response.jsx

```diff
@@ -264,13 +293,40 @@ class Response extends React.Component {
               <Select
                 className={classes.select}
                 onChange={this.onChange('validation')}
-                value={validation}
+                value={validation || 'literal'}
               >
                 <MenuItem value="literal">Literal Validation</MenuItem>
-                {/* Remove symbolic validation PD-690 <MenuItem value="symbolic">Symbolic Validation</MenuItem>*/}
+                <MenuItem value="symbolic">Symbolic Validation</MenuItem>
               </Select>
             </InputContainer>
           </div>
+          {validation === 'literal' && (
+            <div className={classes.flexContainer}>
+              {cAllowTrailingZeros.enabled && (
+                <FormControlLabel
+                  label={cAllowTrailingZeros.label}
+                  control={
+                    <Checkbox
+                      checked={allowTrailingZeros}
+                      onChange={this.onAllowTrailingZerosChange}
```

Comment:
So here you'll have something like:
e => this.onCheckboxChange(e, 'allowTrailingZeros')

Reply:
done